### ISSUE TITLE
April 28 - Accessibility + Frontend Polish

### ISSUE THREAD

**Issue by ChasVanDav** [ChasVanDav/Astute-Abroad#15]
- [x] Add ARIA labels and keyboard navigation support
- [x] Check color contrast and fix accessibility warnings
- [x] Add client-side form validation to login, input, search
- [x] Test on mobile and multiple screen sizes

**Comment by mai-repo** [comment]
@ChasVanDav, for the login page, it would be good to add labels; it seems that WAVE is pointing that out as something missing.  Nice work making sure the color contrast is accessible and easy for everyone to read.

**Comment by ChasVanDav** [comment]
> [@ChasVanDav](https://github.com/ChasVanDav), for the login page, it would be good to add labels; it seems that WAVE is pointing that out as something missing. Nice work making sure the color contrast is accessible and easy for everyone to read.

Adding this to my to-do list! TY

**Comment by ChasVanDav** [comment]
issue resolved by pr #95!
